Livecheckables/musepack.rb

```diff
@@ -0,0 +1,6 @@
+class Musepack
+  livecheck do
+    url "https://www.musepack.net/index.php?pg=src"
+    regex(/href=.*?musepack[._-]src[._-]v?(r\d+)\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?musepack[._-]src[._-](r\d+)\.t/i)
```

Based on the file name format, I don't imagine we'll encounter a file name like `musepack_src_vr123.tar.gz`. I'm fine with omitting the`v?` here, as it doesn't use a common version scheme.